app/views/travel_advice/index.html.erb

```diff
@@ -3,55 +3,71 @@
-            <li><%= link_to "feed", travel_advice_path(@presenter.slug, format: "atom"), :class => "feed", :title => "Subscribe via RSS" %></li>
-          </ul>
-        </section>
+  <div class="travel-container js-travel-container">
+    <section class="govuk-grid-row">
+      <div id="country-filter" class="govuk-grid-column-one-half">
+        <form class="country-filter-form">
+          <fieldset class="country-filter-form__form-group">
+            <%= render "govuk_publishing_components/components/input", {
+              label: {
+                text: "Search for a country or territory"
+              },
+              id: "country",
+              name: "country",
+              type: "text",
+              search_icon: true,
+              width: 20,
+            } %>
+          </fieldset>
+        </form>
+      </div>
+      <div class="govuk-grid-column-one-half subscriptions-wrapper">
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-1">Get updates</h2>
+        <%= render "govuk_publishing_components/components/subscription-links", {
+          email_signup_link_text: "email",
+          email_signup_link: @presenter.subscription_url,
+          feed_link_text: "feed",
+          feed_link: travel_advice_path(@presenter.slug, format: "atom"),
+          hide_heading: true,
+          margin_bottom: 0,
+        } %>
       </div>
-      <section class="countries-wrapper">
-        <div class="countries">
-          <h1>Countries or territories</h1>
-          <p class="country-count">
-            <span class="js-filter-count"><%= @presenter.countries.length %></span>
-            <span class="visuallyhidden">Countries or territories</span>
-          </p>
-        </div>
+      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible app-full-width-break">
+    </section>
 
-        <div class="countries-list">
-          <% @presenter.countries_grouped_by_initial_letter.each do |initial,countries| %>
-            <div id="<%= initial %>" class="list">
-              <h2><span class="visuallyhidden">Countries starting with </span><%= initial %></h2>
-              <ul class="countries">
-                <% countries.each do |country| %>
-                  <li data-synonyms='<%= country.synonyms ? country.synonyms.join("|") : "" %>'><%= link_to country.name, "/foreign-travel-advice/#{country.identifier}", class: "govuk-link" %></li>
-                  <% end %>
-              </ul>
-            </div>
-          <% end %>
-        </div>
-      </section>
-    </div>
+    <section class="js-countries-wrapper countries-wrapper govuk-grid-row">
+      <div class="countries govuk-grid-column-one-third">
+        <h2 class="govuk-heading-m">Countries or territories</h2>
+        <p class="country-count js-country-count">
+          <span class="js-filter-count filter-count "><%= @presenter.countries.length %></span>
+          <span class="visuallyhidden">Countries or territories</span>
```

Comment:
`govuk-visually-hidden` ?

Reply:
There's always one that gets away...

Changed to `govuk-visually-hidden`.